hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -38,6 +39,7 @@ def _manage_gauge(self, metric, formatted_query_result):
             metric (dict): a dictionary containing information about the metric
             formatted_query_result (nested list): query formated by _format_query_result method
         """
+        logger = logging.getLogger(__name__)
```

Comment:
I would create self._logger parameter during the initialization instead of just adding this line every method. It looks like a better approach